tests/test_app.py

```diff
@@ -1236,7 +1245,7 @@ async def test_bulk_specification_dts_fail_wrong_extension():
                     {
                         "type": "cannot_parse_file",
                         "file": f"{dts_dir}/{manifest}",
-                        "message": "foo is not a supported file type for import specifications",
+                        "message": f"{expected} is not a supported file type for import specifications",
```

Comment:
Error is a little funky for the no extension case, but should be understandable

Reply:
The other option is rewriting a bunch of things to handle something that'll probably (hopefully) be pretty rare.